markdeck

```diff
@@ -4,6 +4,7 @@ set -ue
 
 BUILD=deck
 VERSION=0.1
+PORT=8080
```

Comment:
But this is still static, isn't it?
I would expect something like `PORT=${1:-8080}` or `PORT=${PORT:-8080}` or the like...
and: gets the correct port number displayed in the mark deck console (I mean at the end of the loop output)?